.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: python
+python: 3.6
+cache: pip
+install:
+  - pip install pipenv
+  - pipenv install --dev
+  - wget https://github.com/Sage-Bionetworks/infra-utils/archive/master.zip -O /tmp/infra-utils.zip
+  - unzip -j -n /tmp/infra-utils.zip -x "infra-utils-master/.gitignore" "infra-utils-master/LICENSE" "infra-utils-master/*.md" "infra-utils-master/aws/*"
+  - ./setup_aws_cli.sh || travis_terminate 1
+  - export SAM_CLI_TELEMETRY=0
+stages:
+  - name: validate
+  - name: test
+    if: branch = master AND type = push
+  - name: deploy
+    if: (tag =~ ^v) OR (branch = master AND type = push)
+jobs:
+  fast_finish: true
+  include:
+    - stage: validate
+      script:
+        - pre-commit autoupdate
+        - pre-commit run --all-files
+        - sam build
+        - python -m pytest tests/ -v
+    - stage: test
+      script:
+        - sam build
+        - |
+          sam package --template-file .aws-sam/build/template.yaml \
```

Comment:
shouldn't the `--template-file` reference the source template not the generated one?

Reply:
I don't think so. This is exactly what we do in all the other lambda builds.

Reply:
I've taken another look and i've verified that my comment is accurate.  We've actually been doing wrong on aws-infra repo.  This only works if you do a `sam build` before `sam package`.  You want to run `sam package` independently and in that case you need to provide the source template. which in this case is just `template.yaml`